src/TaylorSeries.jl

```diff
@@ -191,13 +191,13 @@ function numbr2str{T<:Real}(zz::Complex{T}, ifirst::Bool=false)
 end
 
 #name_taylorNvar(n::Int) = string("⋅x", subscriptify(n))
-name_taylorNvar(i::Int) = string(" ", _params_taylorN.variable_names[i])
+name_taylorNvar(i::Int) = string(" ", _params_TaylorN_.variable_names[i])
```

Comment:
To be fully consequent with this PR, shouldn't we add the following new function?
```julia
function get_variable_names(i::Int)
    @assert 1 ≦ i ≦ get_numvars()
    _params_TaylorN_.variable_names[i]
end
get_variable_names() = _params_TaylorN_.variable_names
``` 
A reason against this is that it would be used only once, as far as I can see...